docs/website/docs/developers/debugging/sanitizers.md

```diff
@@ -39,6 +39,65 @@ Enabling ASan in the IREE build is a simple matter of setting the
 cmake -DIREE_ENABLE_ASAN=ON ...
 ```
 
+#### Linking to the dynamic ASan runtime
+
+You may want to use ASan when using the python bindings.
+One way to achieve this is to build Python with ASan or whatever executable is
+going to use IREE as a shared library.
+Another option is to link to the ASan runtime dynamically instead of
+linking it statically into an executable.
+
+```shell
+cmake \
+  -DIREE_ENABLE_ASAN=ON \
+  -DCMAKE_EXE_LINKER_FLAGS=-shared-libasan \
+  -DCMAKE_SHARED_LINKER_FLAGS=-shared-libasan \
+  -DCMAKE_C_COMPILER=clang-12 \
```

Comment:
We just need to specify this is a clang? No need to be at the exact version? So something like `-DCMAKE_C_COMPILER=$(which clang)`?

Or maybe clarify at the beginning saying we are using clang-12 as an example. (Given in the following you are using clang-12 paths.)

Reply:
I added explanation that this is a clang-12 example. I am not using a version general example, since the ASan runtime library path is different in other packages and the package names may be different.